### PR TITLE
DOC: added all projects toctree to bluesky

### PR DESCRIPTION
**Pull request by mrakitin**
## Description

Add all projects toctree to the bluesky documentation.

## Motivation and Context

Motivated by https://github.com/NSLS-II/docs/issues/73.

## How Has This Been Tested?

Tested locally by building the documentation.

### CODE REVIEW

File: doc/source/index.rst [hunk 1]
```diff
@@ -60,3 +60,34 @@ Index
...
+   databroker <https://nsls-ii.github.io/databroker>
+   amostra <https://nsls-ii.github.io/amostra>
+   datamuxer <https://nsls-ii.github.io/datamuxer>
+   suitcase <https://nsls-ii.github.io/suitcase>
+
+.. toctree::
+   :hidden:
+   :caption: Data Analysis
+
+   scikit-beam <http://scikit-beam.github.io/scikit-beam>
```
Comment (danielballan) [inline doc/source/index.rst]: Should we include this one? It sort of breaks the "domain-agnostic" vibe that we're pushing in the data acquisition stack. Not sure how important that is.

File: doc/source/index.rst [hunk 2]
```diff
@@ -60,3 +60,34 @@ Index
...
+
+.. toctree::
+   :hidden:
+   :caption: GitHub Links
+
+   NSLS-II Repositories <https://github.com/NSLS-II/>
+   Wish List <https://github.com/NSLS-II/wishlist/issues>
```
Comment (danielballan) [inline doc/source/index.rst]: This wishlist repo is basically not used. Before we copy the link across N repos: do we still _want_ to use it?

Comment (prjemian) [reply doc/source/index.rst]: The wish list is most useful from the main page (http://nsls-ii.github.io/).  The content that appears across all *N* repos is most useful from mobile clients where editing the address URL can be cumbersome.

File: doc/source/index.rst (hunk 1 above)
Comment (prjemian) [reply doc/source/index.rst]: scikit-beam has a different style, indicating it is outside of the standard stack, just link to it from the top page (http://nsls-ii.github.io/)

Comment (mrakitin) [reply doc/source/index.rst]: Agree. Will keep it in the docs repo, but will remove from the project-specific documentation.

File: doc/source/index.rst (hunk 2 above)
Comment (mrakitin) [reply doc/source/index.rst]: Agree with that too. I'll remove the wish list link from the project-specific docs, leaving it at the main page.

**Pull request merged**